src/gui/mzroll/mainwindow.cpp

```diff
@@ -903,18 +905,18 @@ AutoSave::AutoSave(MainWindow* mw)
 {
     _mainwindow = mw;
     _mainwindow->timestampFileExists = false;
-    saveTablesOnly = false;
+    singleGroupToBeSaved = nullptr;
 }
 
-void AutoSave::saveProjectWorker(bool tablesOnly)
+void AutoSave::saveProjectWorker(PeakGroup *singleGroup)
 {
-    saveTablesOnly = tablesOnly;
+    singleGroupToBeSaved = singleGroup;
```

Comment:
@saifulbkhan Turns out I can batch select a few groups and mark them as good or bad. What would the behavior be in that case?

Reply:
We can? I did not know about that. What would probably happen is that only one the groups would actually be saved with the g/b labels. But I'll make sure this is handled correctly.